pkg/cloudprovider/providers/azure/azure.go

```diff
@@ -51,10 +51,13 @@ const (
 	CloudProviderName      = "azure"
 	rateLimitQPSDefault    = 1.0
 	rateLimitBucketDefault = 5
-	backoffRetriesDefault  = 6
-	backoffExponentDefault = 1.5
-	backoffDurationDefault = 5 // in seconds
-	backoffJitterDefault   = 1.0
+	// VMSS API limits are 180 per 3 minutes & 900 per 30 minutes
```

Comment:
this PR would add the cloud provider rate limit, and `180 per 3 minutes & 900 per 30 minutes` are for the whole subscription. It's hard to tell the correct rate limit value per vmss cluster.